### ISSUE TITLE
Fetal lower urinary tract obstruction

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7195]
**Preferred term label:**
Fetal lower urinary tract obstruction

Question -- is this distinct from the existing term [Urethral obstruction HP:0000796](https://hpo.jax.org/app/browse/term/HP:0000796)? Should there be a new HPO term for this?

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Probably this is a valid term.  Look for PMIDs etc.

**Comment by pnrobinson** [comment]
done
